apollo-gradle-plugin/src/test/groovy/com/apollographql/apollo/gradle/integration/BasicAndroidSpec.groovy

```diff
@@ -188,6 +188,30 @@ class BasicAndroidSpec extends Specification {
         "return Currency.class;")
   }
 
+  def "adding a custom singularization rule to the build script re-generates the CustomType class"() {
+    setup: "a testProject with a previous build and an apollo extension appended"
+    replaceTextInFile(new File("$testProjectDir/build.gradle")) {
+      it.replace("apollo {", "apollo {\n customSingularizationRules=[\"F(ilm)s\\\$\" : \"SingularF\\\$1\"]\n")
+    }
+
+    System.err.println(new File("$testProjectDir/build.gradle").getText('UTF-8'))
```

Comment:
nit: looks like should be removed?

Reply:
Oof...yes, will remove, sorry about that